### DISCUSSION TITLE
Clerk to be used for allowing my customers to get provided API keys to use our REST API?

### DISCUSSION THREAD

**Discussion by lancejpollard** [clerk/javascript#3078] [Support request - DO NOT OPEN HERE]
If I want to have my customers be able to use my REST API, I need to give them API keys. Do you support this sort of workflow/system at Clerk? Or if not, are there any Clerk-like startups which abstract away the management of API keys like this that you're aware of?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
